doc/writing.md

```diff
@@ -236,9 +234,9 @@ preview panel, and the text will automatically update:
 
 ## YAML Front-Matter
 
-[YAML](http://yaml.org/) front-matter allows to add extra information (metadata)
-to your documents, such as variables for the different templates. Monod expects
-a unique (optional) YAML front-matter section starting at the first line of the
+[YAML](http://yaml.org/) front-matter allows addition of metadata 
+(such as variables for different templates) to the documents. Monod expects
```

Comment:
Sounds good to me. Your sentence framing seems better. Will surely update the change.